src/gpu/generic/sycl/README.md

```diff
@@ -47,50 +92,77 @@ The implementation supports both forward and backward directions.
 `gelu_tanh`, `hardsigmoid`, `hardswish`, `linear`, `log`, `logistic`, `mish`,
 `pow`, `relu`, `round`, `soft_relu`, `sqrt`, `square`,`swish` and `tanh`
 * Supported formats: `NCDHW`, `NDHWC`, `NCHW`, `NHWC`, `NCW`, `NWC`, `NC`, `N`
+* Supported data types: `f32`, `bf16`, `f16`, `s32`, `s8`, `u8`
+
+## Layer Normalization
+
+The implementation supports both forward and backward directions.
+
+* Supported formats: `NCDHW`, `NDHWC`, `NCHW`, `NHWC`, `NCW`, `NWC`, `NC`
+* Supported input/output data types for forward direction: `f32`, `bf16`, `f16`,
+  `s8`, `u8`
+* Supported input/output data types for backward direction: `f32`, `bf16`
+* Supported scale/shift data types: `f32`, `bf16`, `f16`
 
 ## LRN
 
 The implementation supports both forward and backward directions.
 
 * Supported formats: `NCDHW`, `NDHWC`, `NCHW`, `NHWC`, `NCW`, `NWC`, `NC`
+* Supported data types: `f32`, `bf16`, `f16`
+
+## Matmul
+
+* Supported formats: plain formats
```

Comment:
Should we mention something about leveraging the cublaslt format when format tag is any?

Reply:
Nevermind, I confused the documentations for NVIDIA and generic vendors :)